actionview/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+*   Add Enum translation methods.
+
+        class Report < ActiveRecord::Base
+        enum :status, [ :confidential, :publicly_available ]
```

Comment:
```suggestion
          enum :status, [ :confidential, :publicly_available ]
```

Reply:
Thanks!
I'll resolve conflicts and fix it.